terraform/modules/cloudfoundry/variables.tf

```diff
@@ -63,14 +63,10 @@ variable "vpc_id" {}
 variable "private_route_table_az1" {}
 
 variable "private_route_table_az2" {}
+# Assuming these are now being defined by terraform
```

Comment:
I think this information is already covered by the fact that the cidr variables don't have defaults 😄